src/components/Episodes/renderUtils.jsx

```diff
@@ -1,29 +1,36 @@
 import React from "react";
 import Episode from "COMPONENTS/Episode/";
+import { TrailersProvider } from "./";
+import UTILS from "SHARED/utils.js";
 
-function renderEpisode(data, index){
+function renderEpisode(data, index, prevTrailer, nextTrailer){
 
 	const { 
 		number,
 		title,
 		synopsis,
-		image,
-		video,
+		thumbnail,
+		trailer,
 	} = data;
 
-	const id = `episode__${number}`;
+	const id = UTILS.convertToSafeString(title, "-");
 
 	return (
-		<Episode
-			id={ id }
-			key={ id }
-			index={ index }
-			number={ number }
-			title={ title }
-			synopsis={ synopsis }
-			image={ image }
-			video={ video }
-		/>
+		<TrailersProvider 
```

Comment:
Yeah, the context definintely needs to be `Trailer` then, otherwise it sounds like we're providing information for more than one trailer - so this needs to be `<TrailerProvider>`